rust/worker/chroma_config.yaml

```diff
@@ -28,6 +28,7 @@ query_service:
             credentials: "Minio"
             connect_timeout_ms: 5000
             request_timeout_ms: 30000 # 1 minute
+            part_size_bytes: 8388608 # 8MB
```

Comment:
maybe "upload_part_size" is clearer?

Reply:
adb50286

I used `upload_part_size_bytes` since it seemed like the convention was to include units, can drop them if you prefer

Reply:
sorry meant upload_part_size_bytes yes